src/api/entitlements.js

```diff
@@ -175,6 +175,31 @@ export class Entitlements {
   ack() {
     this.ackHandler_(this);
   }
+
+  /**
+   * Converts the whole set of entitlements into a single SKU.
+   * Returns the first SKU that enables this page.
+   * TODO: come up with a better way of picking the right SKU when there are
+   * multiple
+   * @return {?string}
+   */
+  getSingleEntitlement() {
```

Comment:
-> getSkuEntitlement  (or something like that)?

Would it make sense for this function to return all the SKUs and let the caller select which of the SKUs to use (then you wouldn't need your TODO)?

Reply:
The whole point is to centralize this bad logic into a single place so if multiple things require it we only need to fix the 1 function instead of all the things that might rely on this bad logic.  I have a specific use case where I need to pick a single SKU and I'd rather the calling code not need to have that logic in it.

Reply:
I've managed to avoid needing this function.